R/cut.R

```diff
@@ -0,0 +1,129 @@
+#' Cut a numeric Crunch variable
+#'
+#' `crunch::cut()` is equivalent to `base::cut()` except that it operates on
+#' Crunch variables instead of in-memory variables. The function divides the range of
+#' `x` into intervals and codes the values in x according to which interval they fall.
+#' The leftmost interval corresponds to level one, the next leftmost to level two and
+#' so on.
```

Comment:
Since we've been working on documentation, let's make this description nicer. I know this is mostly copied from base::cut, but that doesn't mean we can't do better :)